From 6caf29836346df6a99bcc4df677a67fea06b70d3 Mon Sep 17 00:00:00 2001
From: Kai.Jiang <Kai.Jiang@freescale.com>
Date: Thu, 16 Sep 2010 18:10:05 +0800
Subject: [PATCH 07/10] P1021MDS:ATM:QE pio initial code for ATM feature

This patch adds QE pio initial code for ATM feature.

Signed-off-by: Liu Yu <yu.liu@freescale.com>

[Extracted from vendor drop QorIQ-NONDPAA-SDK-V1-20110429_ltib.iso
linux-2.6.35-qoriq-P1021MDS-ATM-Init-ATM-pio.patch
]

Integrated-by: Jiang Lu <lu.jiang@windriver.com>
---
 arch/powerpc/platforms/85xx/mpc85xx_mds.c |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/platforms/85xx/mpc85xx_mds.c b/arch/powerpc/platforms/85xx/mpc85xx_mds.c
index c7f572f..f111013 100644
--- a/arch/powerpc/platforms/85xx/mpc85xx_mds.c
+++ b/arch/powerpc/platforms/85xx/mpc85xx_mds.c
@@ -227,6 +227,14 @@ static void __init mpc85xx_mds_setup_arch(void)
 
 		for_each_node_by_name(ucc, "ucc")
 			par_io_of_config(ucc);
+
+#if defined CONFIG_ATM_UCC
+{
+		struct device_node *upc;
+		for_each_node_by_name(upc, "upc")
+			par_io_of_config(upc);
+}
+#endif
 	}
 
 	if (bcsr_regs) {
-- 
1.7.0.2

